packages/ui/src/theme/types.ts

```diff
@@ -24,7 +23,7 @@ export type Override =
   | ColorModeOverride;
 
 interface BaseOverride {
-  tokens?: PartialDeep<Tokens>;
+  tokens?: Tokens;
```

Comment:
Does this impact the existing `theme` usage?

Reply:
Good catch!